js/README.md

```diff
@@ -202,11 +197,33 @@ The JS community is a diverse group with a varied list of target environments an
 
 If you think we missed a compilation target and it's a blocker for adoption, please open an issue.
 
+# Community
+
+Full list of broader Apache Arrow [committers](https://arrow.apache.org/committers/) and [projects & organizations](https://github.com/apache/arrow/blob/master/site/powered_by.md).
+
+## Committers
+
+* Brian Hulette, CCRI
+* Paul Taylor, Graphistry, Inc.
```

Comment:
I would not list contributors/committers or their affiliations in the README here. In ASF projects, contributors participate as individuals without any explicit corporate affiliation https://community.apache.org/projectIndependence.html#apache-projects-are-independent

Reply:
I was following Spark's precedent here: http://spark.apache.org/committers.html . Are they in violation?

Reply:
Ah, we have http://arrow.apache.org/committers/ -- it seems it is a convention to list orgs on the website (see also http://hadoop.apache.org/who.html). We should update that page to reflect as a matter of transparency: https://issues.apache.org/jira/browse/ARROW-1912